docker/alpine/Dockerfile

```diff
@@ -0,0 +1,30 @@
+FROM alpine:3.7 as source-builder
+ARG commit
+RUN apk add --no-cache abuild acct alpine-sdk attr autoconf automake bash \
+    binutils binutils-libs bison bsd-compat-headers build-base \
+    c-ares c-ares-dev ca-certificates cryptsetup-libs curl \
+    device-mapper-libs expat fakeroot flex fortify-headers g++ gcc gdbm \
+    git gmp isl json-c json-c-dev kmod lddtree libacl libatomic libattr \
+    libblkid libburn libbz2 libc-dev libcap libcurl libedit libffi libgcc \
+    libgomp libisoburn libisofs libltdl libressl libssh2 \
+    libstdc++ libtool libuuid linux-headers lzip lzo m4 make mkinitfs mpc1 \
+    mpfr3 mtools musl-dev ncurses-libs ncurses-terminfo ncurses-terminfo-base \
+    patch pax-utils pcre perl pkgconf python2 python2-dev readline \
+    readline-dev sqlite-libs squashfs-tools sudo tar texinfo xorriso xz-libs \
+    groff gzip bc
```

Comment:
Build-deps changed yesterday, we now depend on the Sphinx documentation generator

Reply:
Thank you, should be fixed now...